packages/flutter/test/material/slider_test.dart

```diff
@@ -626,252 +625,6 @@ void main() {
     log.clear();
   });
 
-  testWidgets('Slider uses the right theme colors for the right components', (WidgetTester tester) async {
```

Comment:
I'm assuming these tests have just been moved?

Reply:
This is a theme test residing `slider_test.dart`, it should be under `slider_theme_test.dart`.
 I'll need to update this test for M3 and M2 groups in a new future PR under `slider_theme_test.dart`

Reply:
OK